README.md

```
@@ -7,8 +7,35 @@
 
 ## Why Lisa?
 
-Unlike simple vector databases or file-based memory, Lisa uses **[Graphiti](https://github.com/getzep/graphiti)** - a knowledge graph that captures *relationships* between concepts, not just text.
+### Because you would like every Claude session to remember the last one, the one from last week, last month. ALL OF THEM!!.
 
+## You want..
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)